test/args-test.js

```diff
@@ -0,0 +1,30 @@
+import { test } from 'ava'
+
+import Args from '../lib/args'
+
+test('encode empty', t => {
+  const args = {}
+  const encoded = Args.encode(args)
+  t.deepEqual(encoded, '%7B%7D')
+})
+
+test('file with query string', t => {
+  const url = '/test/file.html?p=hello&n=world'
+  const args = {}
+  const newUrl = Args.urlWithArgs(url, args)
+  t.deepEqual(newUrl, 'file:///test/file.html?p=hello&n=world#%7B%7D')
```

Comment:
You know what, I see how that is being invoked so this is fine for now.
I will merge this asap